src/lib/tvgRender.h

```diff
@@ -36,6 +37,11 @@ struct Surface
     uint32_t cs;
 };
 
+struct Composite {
+    void* compEData;
```

Comment:
compEData is too abstract. Please append some descriptive comment for this variable.

Reply:
in the end, my suggestion is 
compEData => edata;

Reply:
Yes I modified it.